docs/model_configuration.md

```diff
@@ -125,9 +125,11 @@ When a model is using the auto-complete feature, a default maximum batch size ma
 be set by using the `--backend-config=default-max-batch-size=<int>` command line argument. 
 This allows all models which are capable of batching and which make use of 
 [Auto Generated Model configuration](#auto-generated-model-configuration) to have
-a default maximum batch size. This value is set to 4 by default. While none of the
-officially supported Triton backends implement this feature, backend developers 
-may make use of this value by obtaining it from the TRITONBACKEND_BackendConfig api.
+a default maximum batch size. This value is set to 2 by default. If the value is
+greater than 1, the dynamic batch scheduler will be set for the model if
```

Comment:
All the talk about enabling dynamic batching should  be a part of #auto-generated-model-configuration section and not here.

Reply:
I'm a little unclear whether you want me to split this documentation up or move the whole paragraph up.

Reply:
I am talking about splitting this documentation. The things with dynamic batcher you are talking about makes more sense in auto complete config.

Reply:
Updated documentation to split out the dynamic batching explanation.